web-components/README.md

```diff
@@ -0,0 +1,68 @@
+# Data Commons JS Web Components
+
+Embed [Data Commons](https://datacommons.org)
+[statistical variable](https://datacommons.org/tools/statvar) observation
+visualizations in your web application.
+
+## Usage
+
+Include datacommons.js and styles in your html `<head>...</head>` tag
+
+```html
+<link rel="stylesheet" href="https://datacommons.org/css/ranking.min.css" />
+<link rel="stylesheet" href="https://datacommons.org/css/nl_interface.min.css" />
+<script src="https://datacommons.org/datacommons.js"></script>
+```
+
+Then add the web components desired. A minimal example would be:
+```html
+<html>
+  <head>
+    <link rel="stylesheet" href="https://datacommons.org/css/ranking.min.css" />
+    <link rel="stylesheet" href="https://datacommons.org/css/nl_interface.min.css" />
+    <script src="https://datacommons.org/datacommons.js"></script>
+  </head>
+  <body>
+    <!-- Embed a line chart -->
+    <datacommons-line
+      title="US Population Over Time"
+      place="country/USA"
+      variables='["Count_Person"]'
```

Comment:
(not blocking for this pr): we should prioritize releasing the space-separated attribute converter

Reply:
+1, once we make the change I'll update the examples here as well.